src/features/student/views/StudentToolsTracesView/components/StudentToolsTracesAddTraceDrawer/components/CreateTraceFormDeclarationItems/CreateTraceFormDeclarationItems.vue

```diff
@@ -86,43 +53,19 @@ function handleToggleChange (fieldName: 'isAuthentic' | 'isGroup' | 'useIA', val
           {{ t('student.views.studentToolsTracesView.studentToolsTracesAddTraceDrawer.createTraceForm.declaration.iaUsage.title') }}
         </p>
 
-        <form.Field name="useIA">
-          <template #default="{ field }">
-            <div class="declaration-items__field">
-              <AvToggle
-                id="useIA"
-                name="useIA"
-                :model-value="field.state.value"
-                :description="t('student.views.studentToolsTracesView.studentToolsTracesAddTraceDrawer.createTraceForm.declaration.iaUsage.label')"
-                :active-text="t('global.avToggle.activeText')"
-                :inactive-text="t('global.avToggle.inactiveText')"
-                @update:model-value="(value) => handleToggleChange('useIA', value, field.handleChange)"
-              />
-              <div
-                v-if="field.state.meta.errors.length > 0"
-                class="declaration-items__error"
-              >
-                {{ field.state.meta.errors.join(', ') }}
-              </div>
-            </div>
-          </template>
-        </form.Field>
-
-        <form.Field
-          v-if="showIAJustification"
-          name="iaJustification"
-        >
-          <template #default="{ field }">
-            <TraceIaJustificationTextarea
-              id="ia-justification"
-              v-model="field.state.value"
-              :error-message="field.state.meta.errors.join(', ')"
-              :required="showIAJustification"
-              @blur="field.handleBlur"
-              @update:model-value="(value) => typeof value == 'string' ? field.handleChange(value) : field.handleChange(undefined)"
-            />
-          </template>
-        </form.Field>
+        <div class="declaration-items__field">
+          <TraceAiUsageToggleFormField
+            id="useIA"
+            :form="form"
+            @change="onChangeAiUsageToggle"
+          />
+        </div>
+
+        <TraceAiJustificationTextareaFormField
+          id="ia-justification"
```

Comment:
```suggestion
          id="iaJustification"
```
C'est le seul id que tu avais en kebab-case, autant rester uniforme